salt/utils/dns.py

```diff
@@ -1216,5 +1204,67 @@ def parse_resolv(src="/etc/resolv.conf"):
             "search": search,
             "options": options,
         }
-    except IOError:
+    except OSError:
+        return {}
+
+
+def parse_systemd_resolve():
+    """
+    Parse the information from systemd-resolve
+    """
+
+    nameservers = []
+    ip4_nameservers = []
+    ip6_nameservers = []
+    domain = ""
+
+    cmd = "systemd-resolve --status"
+    cmd = __salt__["cmd.run_all"](
+        "{}".format(cmd), python_shell=False, output_loglevel="quiet"
+    )
+
+    if cmd["retcode"] == 0:
+        prev_section = None
+        lines = [re.sub("[\t ]", "", line) for line in cmd["stdout"].splitlines()]
```

Comment:
I do not know the output for an example, but my guess is that `systemd-resolve` doesn't support a parseable output, nor a plain `.strip()` would do it?
Wouldn't `.replace("\t", "").replace(" ", "")` achieve the same? Or is `re.sub` just faster?

Reply:
Unfortunately it doesn't have an option to output into a parseable format like JSON.  I'll test with using `.replace` and see how that works.